File: S3Demo.Web/S3Demo.Model/Reading.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace S3Demo.Model
{
    public partial class Reading
    {
        public int BuildingId { get; set; }
        public int ObjectId { get; set; }
        public int DataFieldId { get; set; }
        public double Value { get; set; }
        public DateTime Timestamp { get; set; }
    }
}

File: S3Demo.Web/S3Demo.Web/ClientApp/src/app/models/reading.ts
export class Reading {
  BuildingId: number;
  ObjectId: number;
  DataFieldId: number;
  Value: number;
  Timestamp: string;
  constructor() {
    this.BuildingId = 0;
    this.ObjectId = 0;
    this.DataFieldId = 0;
    this.Value = 0;
    this.Timestamp = '00:00';
  }
}
